### PR TITLE
CFE-271: External DNS GA

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): OCP 4.10+

Issue: https://issues.redhat.com/browse/CFE-271

Link to docs preview: https://gabriel-rh.github.io/PR46937/PR46937/networking/external_dns_operator/nw-creating-dns-records-on-aws.html

Additional information:

### CODE REVIEW

File: modules/nw-control-dns-records-public-hosted-zone-aws.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="nw-control-dns-records-public-hosted-zone-aws_{context}"]
 = Creating DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator
 
-You can create DNS records on a public hosted zone for AWS by using the Red Hat External DNS Operator.
+You can create DNS records on a public hosted zone for AWS by using the Red Hat External DNS Operator. You can use the same instructions to create DNS records on a public hosted zone for AWS GovCloud.
```
Comment (alebedev87) [inline modules/nw-control-dns-records-public-hosted-zone-aws.adoc:9]: I believe AWS GovCloud's Route53 still doesn't support public hosted zones. So, better to not mention `public` for GovCloud.

Comment (xenolinux) [reply modules/nw-control-dns-records-public-hosted-zone-aws.adoc]: Addressed- Thanks for the review!

**Comment by quarterpin** [comment]
/lgtm 
thanks!

**Comment by quarterpin** [comment]
@xenolinux, one more point to be mentioned, AWS govcloud support is added as a "techpreview" in the current release of ExternalDNS operator. Rest looks fine.

File: networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc [hunk 2]
```diff
@@ -6,6 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can create DNS records on AWS using External DNS Operator.
+You can create DNS records on AWS and AWS GovCloud using External DNS Operator.
```
Comment (maxwelldb) [inline networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc:9]:
```suggestion
You can create DNS records on AWS and AWS GovCloud by using External DNS Operator.
```

**Review by maxwelldb — COMMENTED** [review]
One suggestion per ISG.

**Pull request merged**